spec/features/borrower_pays_back_loan_spec.rb

```diff
@@ -0,0 +1,59 @@
+require "rails_helper"
+
+RSpec.feature "borrower pays back loan" do
+  let!(:lender) { User.create(email: "jorge@example.com",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.